exporters/otlp/otlpmetric/otlpmetrichttp/client_test.go

```diff
@@ -192,6 +192,25 @@ func TestConfig(t *testing.T) {
 		assert.Len(t, rCh, 0, "failed HTTP responses did not occur")
 	})
 
+	t.Run("WithRetryAndExporterErr", func(t *testing.T) {
```

Comment:
Should this test be added to log and trace too?

Reply:
Sorry, it's a miss from my side. Traces already had a test case for the use case there I have changed how the error message will reflect now. Added for logs.